partners.yml

```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Unique identifier of the bank account received from the Send Tink Code endpoint.
```

Comment:
I would like to rephrase it a little bit, so the understand what is the usage of it 🤔  Does something like the following makes sense for you? 🤔 
`Unique identifier of the bank account for the payout chosen by the customer from the list of accounts returned by the "/bank/send-code" endpoint`

Reply:
Yes that seems to be more descriptive!